### PR TITLE
EnC - Parameterless struct constructors

### PR DESCRIPTION
**Pull request by davidwengier**
Fixes https://github.com/dotnet/roslyn/issues/55547

Feels too easy, probably missing something ¯\\\_(ツ)_/¯

### CODE REVIEW

**Review by ryzngard — APPROVED** [review]
Test only change? LGTM :shipit:

File: src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs [hunk 1]
```diff
@@ -1438,12 +1442,43 @@ static void Main(string[] args)
...
+    }
+}";
+            var edits = GetTopEdits(src1, src2);
+            var active = GetActiveStatements(src1, src2);
+
+            edits.VerifyRudeDiagnostics(active);
+        }
+
+        [Theory]
+        [InlineData("class ")]
```
Comment (tmat) [inline src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs:1463]: >  [](http://example.com/codeflow?start=26&length=1)

extra space

**Review by tmat — APPROVED** [review]
:shipit:

**Comment by tmat** [comment]
Do we have test for parameterless ctor in records and record structs? Might be good to add one.

BTW, noticed https://github.com/dotnet/roslyn/issues/52563 is closed by still references in the source code.

**Comment by davidwengier** [comment]
> BTW, noticed #52563 is closed by still references in the source code.

Whoops, thanks.

**Pull request merged**